component/exporter.go

```diff
@@ -56,7 +56,7 @@ type ExporterCreateParams struct {
 	ApplicationStartInfo ApplicationStartInfo
 }
 
-// ExporterFactory can create TracesExporter and MetricsExporter. This is the
+// ExporterFactory can create TracesExporter, MetricsExporter and LogsExporter. This is the
```

Comment:
```suggestion
// ExporterFactory can create TracesExporter, MetricsExporter, and LogsExporter. This is the
```

Never not Oxford comma!